pkg/apis/core/validation/validation.go

```diff
@@ -65,7 +65,7 @@ const csiDriverNameRexpErrMsg string = "must consist of alphanumeric characters,
 const csiDriverNameRexpFmt string = `^[a-zA-Z0-9][-a-zA-Z0-9_.]{0,61}[a-zA-Z-0-9]$`
 
 var pdPartitionErrorMsg string = validation.InclusiveRangeError(1, 255)
-var fileModeErrorMsg string = "must be a number between 0 and 0777 (octal), both inclusive"
+var fileModeErrorMsg = "must be a number between 0 and 0777 (octal), both inclusive"
```

Comment:
is there a reason to drop this type declaration and not the others around it?

Reply:
@liggitt linter give warning only on that declaration , may be its due to the presence of `"` in right hand side which explicitly make sure its a string. no warning I could see on other declarations.

Reply:
I don't see a lot of value in sweeping to satisfy random golint complaints that don't make a lot of sense. @dims, can we take a step back on https://github.com/kubernetes/kubernetes/issues/68026 and figure out if we can scope the golint warnings and things we're fixing down to issues that are actually impactful (things like missing/nonstandard docs, etc)